build-system/babel-config/minified-config.js

```diff
@@ -11,7 +11,7 @@ const {getReplacePlugin} = require('./helpers');
  * @return {!Object}
  */
 function getMinifiedConfig() {
-  const isProd = argv._.includes('dist') && !argv.fortesting;
+  const isProd = BUILD_CONSTANTS.IS_PROD;
```

Comment:
Suggestion: In cases like this where there's only one usage spot, why not remove the `isProd` variable and directly use the constant? Might make the code easier to understand.